### PR TITLE
fix: add hyphen on messageLint match

### PR DESCRIPTION
**Pull request by shinjiescorido**

### CODE REVIEW

**Comment by shinjiescorido** [comment]
@openam

File: test/test-files/messageType/messageType1.js [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 module.exports = {
 	'config' : {
-		'messageType' : 'v1Test'
+		'messageType' : 'v1.test-version.create'
```
Comment (yamii) [inline test/test-files/messageType/messageType1.js:5]: can you also add test for the ff: 

```
v1.testVersion-minor-.create
V1.test-version.create
v1.test-version-create
v1.testVersion.create
v1.test.createVersion
```

?

**Comment by yamii** [comment]
@shinjiescorido please add 
`Issue <StoryNumber>` to your commit after your commit message

File: test/test-files/messageType/messageType1.js (hunk 1 above)
Comment (shinjiescorido) [inline test/test-files/messageType/messageType1.js:5]: the new messageTypes were added at the next 'describe' method. [link](https://github.com/shinjiescorido/pathlint/blob/54b82a40190ef8ef161b089a55dfc4bfcd6a4742/test/adapters/message-lint.js#L30)

**Comment by openam** [comment]
The commit message should be:

```
fix: add hyphen on messageLint match

Issue S-06698
```

See https://github.com/sinet/coding-conventions/blob/master/git.md#message-footer

**Comment by shinjiescorido** [comment]
done changing the commit message. thank you

File: test/test-files/messageType/messageType1.js (hunk 1 above)
Comment (yamii) [inline test/test-files/messageType/messageType1.js:5]: thanks!

**Comment by sydcanem** [comment]
👍

File: lib/adapters/message-lint.js [hunk 2]
```diff
@@ -20,7 +20,11 @@ module.exports = function messageLint ( args ) {
 						.split( ',' );
 
 		_.forEach( messageType, function ( message ) {
-			var isBaseNameGood = args.pattern.test( message );
+			var isBaseNameGood = args.pattern.test( message )
+				// check if camel case
+				&& !message.match( /[A-Z]/g )
+				// check for . character
+				&& ( message.match( /\./g ) || [] ).length === 2;
```
Comment (openam) [inline lib/adapters/message-lint.js:27]: This is very specific to our lint rules.

File: lib/adapters/message-lint.js [hunk 3]
```diff
@@ -20,7 +20,11 @@ module.exports = function messageLint ( args ) {
 						.split( ',' );
 
 		_.forEach( messageType, function ( message ) {
-			var isBaseNameGood = args.pattern.test( message );
+			var isBaseNameGood = args.pattern.test( message )
+				// check if camel case
+				&& !message.match( /[A-Z]/g )
```
Comment (openam) [inline lib/adapters/message-lint.js:25]: This is very specific to our lint rules.

**Comment by openam** [comment]
@shinjiescorido you skipped the blank line before the footer.

**Pull request merged**